app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt

```diff
@@ -85,103 +92,24 @@ fun HomeScreenContent(title: String) {
                 .size(dimensionResource(id = R.dimen.margin_padding_size_xxlarge))
         )
         Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
-        SearchBar()
-        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.margin_padding_size_medium)))
-
-        Column(
-            modifier = Modifier
-                .fillMaxSize()
-                .verticalScroll(rememberScrollState())
+        AnimatedVisibility(
```

Comment:
based on the previous comments , this AnimatedVisibility should enclose the Text, Spacer, Image, Spacer that are above this line , and not the searchbar